core/codegen/src/derive/form_field.rs

```diff
@@ -291,5 +292,39 @@ pub fn validators<'v>(
             })).unwrap()
         });
 
-        Ok(exprs)
+    Ok(exprs)
+}
+
+pub fn defaults<'v>(
+    field: Field<'v>,
+) -> Result<syn::Expr> {
```

Comment:
This should be called `default`; there can only be one default. It should error if there is more than one default specified.